File: python/String-2/end_other.py
def end_other(a, b):
  checker = ""
  if(len(a) >= len(b)):
    i = -len(b)
    while(i < 0):
      checker += a[i].lower()
      i += 1
    return checker == b.lower()
  else:
    i = -len(a)
    while(i < 0):
      checker += b[i]
      i += 1
    return checker.lower() == a.lower()

File: java/String-2/endOther.java
public boolean endOther(String a, String b) {
	if(a.length() < b.length()) {
		return b.substring(b.length()-a.length(), b.length()).toLowerCase().equals(a.toLowerCase());
	}
	return a.substring(a.length()-b.length(), a.length()).toLowerCase().equals(b.toLowerCase());
}
